python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -826,6 +826,28 @@ def ElementwiseMulTensorIntModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwiseMishModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.mish(x)
+
+
+@register_test_case(module_factory=lambda: ElementwiseMishModule())
+def ElementwiseMishModule_basic(module, tu: TestUtils):
+    module.forward(2 * tu.rand(5, 3) - 0.5)
```

Comment:
nit: use `low=` and `high=` to specify range for `tu.rand`. See: https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#things-to-consider-when-creating-new-tests

Reply:
Hi @ramiro050 , so I referred the said link and in the example [above](https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#test-case-function) I see the use of `tu.rand` similar to the way I used here in the PR and other test cases we see in `elementwise.py` - since we were to follow similar convention/style of the file we're adding the test case into, I tried doing the same.

Most of the `tu.rand` invocations we see in the test file are using the default value for `low` and `high` parameters.

I believe unless we use the non-default values we shouldn't have to explicitly mention the parameters and their values while invoking the function - a similar invocation of `tu.randint` can be seen in the example [here](https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#test-case-function).

Let me know if a change is still warranted else we can aim to get this PR in.

Reply:
Hi @Abhishek-Varma, yeah this file contains some of earliest tests added to torch-mlir before there was support for specifying `low` and `high` in `tu.rand`, so people resorted to multiplying and subtracting to change the range. However, for new tests, it is better to use `low` and `high` than the external linear transformation -- `2 * x - 0.5` in this case -- since it makes it much easier to tell what range is being tested.

Reply:
I understand, thanks @ramiro050 - I've addressed this now.
You may check and approve/initiate the workflow if all looks good. :)